docs/topics/modules/e2e/analyze_che_dependencies.adoc

```diff
@@ -0,0 +1,30 @@
+[#analyze_che_dependencies]
+= Analyzing Che Dependencies
+
+The Che workspace automatically analyzes dependencies for the code. To see how this analyzer works:
+
+. Return to the Che workspace browser tab.
+. In the `Project Explorer` panel, double click the `pom.xml` file.
++
+image::pomxml.png[Pom.xml File]
++
+. In the file, go to line 43 and add a line break.
+. Add the following XML code to the file after the line break:
++
+[source,xml]
+----
+<dependency>
+  <groupId>ch.qos.logback</groupId>
+  <artifactId>logback-core</artifactId>
+  <version>1.1.0</version>
+</dependency>
+----
++
+. An error icon appears near the line numbers when you add the XML code. Mouse over the icon to view the error message (see screen capture).
++
+image::red_x.png[Error Icon]
++
+The dependency analyzer displays this error to indicate that the current version (1.1.0) has a CVE (Common Vulnerabilities and Exposures). It also recommends using version 1.2.1 instead to fix the problem.
```

Comment:
"has a CVE" would probably be better to expand a little, such as "has a CVE issue associated with it".